x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/ingest/SetSecurityUserProcessor.java

```diff
@@ -93,7 +99,11 @@ public IngestDocument execute(IngestDocument ingestDocument) throws Exception {
             switch (property) {
                 case USERNAME:
```

Comment:
This makes it so that the user can specify the `username` property when defining the pipeline, but the field is actually named `name` (in 8 when `ecs_compliat` is `true` by default). This is mildly confusing. I would suggest that we add a new property `name` with the same value as `username`.